lib/src/main/java/formflow/library/utils/HashMapConverter.java

```diff
@@ -0,0 +1,41 @@
+package formflow.library.utils;
+
+import com.fasterxml.jackson.core.JsonProcessingException;
+import com.fasterxml.jackson.databind.ObjectMapper;
+
+import javax.persistence.AttributeConverter;
+import java.io.IOException;
+import java.util.HashMap;
+import java.util.Map;
+
```

Comment:
I can add some comments tomorrow - I borrowed some of this idea from a website which I can credit as well.